learning/tour-of-beam/frontend/lib/locator.dart

```diff
@@ -35,11 +35,30 @@ import 'state.dart';
 final _client = CloudFunctionsTobClient();
 
 Future<void> initializeServiceLocator() async {
+  await _initializeRepositories();
   _initializeAuth();
   _initializeState();
   _initializeCaches();
 }
 
+Future<void> _initializeRepositories() async {
+  final routerUrl = await getRouterUrl();
+  final runnerUrls = await waitMap({
+    for (final sdk in Sdk.known) sdk.id: getRunnerUrl(sdk),
```

Comment:
You don't need Scala here. Furthermore this is a dependency that can break the app.
The easiest way is to hardcode the 3 SDKs of ToB.
A better way could be to first call the cloud function to get the SDKs and then do this only for them.

I suggest hardcoding since we aim to hide runners in the next phase so it is only a temporary thing. Add a TODO though to remove that hardcoding if we hide runners.

Reply:
@alexeyinkin SCIO is present in getSdk